Comando,Posição,Direção,Sensor Esquerdo,Sensor Frente,Sensor Direito,Compartimento
LIGAR,"(0, 1)",2,PAREDE,VAZIO,PAREDE,SEM CARGA
INFO,"(0, 1)",2,Iniciando busca pelo humano,SEM CARGA
A,"(1, 1)",2,VAZIO,VAZIO,PAREDE,SEM CARGA
A,"(2, 1)",2,PAREDE,VAZIO,PAREDE,SEM CARGA
A,"(3, 1)",2,PAREDE,VAZIO,PAREDE,SEM CARGA
A,"(4, 1)",2,PAREDE,VAZIO,PAREDE,SEM CARGA
A,"(5, 1)",2,PAREDE,PAREDE,PAREDE,SEM CARGA
ERRO,"(5, 1)",2,"Erro ao executar comando A: Colisao com parede! Tentativa de mover de (5, 1) para (6, 1)",SEM CARGA
ERRO,"(5, 1)",2,"Colisao com parede! Tentativa de mover de (5, 1) para (6, 1)","Traceback (most recent call last):
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\robo_resgate.py"", line 31, in executar_resgate
    self.buscar_humano()
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\robo_resgate.py"", line 72, in buscar_humano
    self.executar_comando('A')
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\robo_resgate.py"", line 168, in executar_comando
    self.simulador.mover_robo(comando)
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\simulador_labirinto.py"", line 72, in mover_robo
    raise ValueError(f""Colisao com parede! Tentativa de mover de {self.posicao_robo} para ({nova_i}, {nova_j})"")
ValueError: Colisao com parede! Tentativa de mover de (5, 1) para (6, 1)
",SEM CARGA
ERRO_FATAL,"(5, 1)",2,"Colisao com parede! Tentativa de mover de (5, 1) para (6, 1)","Traceback (most recent call last):
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\main.py"", line 24, in main
    robo.executar_resgate()
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\robo_resgate.py"", line 31, in executar_resgate
    self.buscar_humano()
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\robo_resgate.py"", line 72, in buscar_humano
    self.executar_comando('A')
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\robo_resgate.py"", line 168, in executar_comando
    self.simulador.mover_robo(comando)
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\simulador_labirinto.py"", line 72, in mover_robo
    raise ValueError(f""Colisao com parede! Tentativa de mover de {self.posicao_robo} para ({nova_i}, {nova_j})"")
ValueError: Colisao com parede! Tentativa de mover de (5, 1) para (6, 1)
",SEM CARGA
